lib/id3c/cli/command/manifest.py

```diff
@@ -483,3 +542,12 @@ def load_ndjson(file: Iterable[str]) -> Iterable:
     Load newline-delimited JSON records from *file*.
     """
     yield from (json.loads(line) for line in file)
+
+
+def find_violating_column_names(extra_columns: List[Tuple[str, Union[str, dict]]]):
```

Comment:
the term "violating" here is a bit vague to me. Could we rename it to something like "protected" or "reserved" to match the global variable? maybe something like "ensure no reserved column names", etc.

Reply:
I went with find_reserved_columns_used_in_extra_column_list